drivers/spi/spi_mcux_flexcomm.c

```diff
@@ -72,7 +70,6 @@ static void spi_mcux_transfer_next_packet(struct device *dev)
 		transfer.txData = (uint8_t *) ctx->tx_buf;
 		transfer.rxData = ctx->rx_buf;
 		transfer.dataSize = ctx->rx_len;
-		transfer.configFlags = 0;
```

Comment:
Isn't there a case we need to set `kSPI_FrameAssert`? I'm not sure I understand why it's always zero.

Reply:
@MaureenHelm there is a case below at line 83
`	if (ctx->tx_count <= 1 && ctx->rx_count <= 1) {
		transfer.configFlags = kSPI_FrameAssert;
	}`

Reply:
I see it now, thanks.